lint/lint.py

```diff
@@ -226,6 +227,31 @@ def check_parsed(repo_root, path, f, css_mode):
     if source_file.type == "visual" and not source_file.name_is_visual:
         return [("CONTENT-VISUAL", "Visual test whose filename doesn't end in '-visual'", path, None)]
 
+    for reftest_node in source_file.reftest_nodes:
+        href = reftest_node.attrib.get("href", "")
+        parts = urlsplit(href)
+        if len(parts.netloc) is not 0:
```

Comment:
also check the scheme (consider `foo:bar`, which has a scheme and a path but no authority)